OptimizelySDK.DemoApp/README.md

```diff
@@ -0,0 +1,50 @@
+# C# SDK Demo App
+
+This demo uses the C# SDK, a part of Optimizely's Full Stack solution. It will walk you through:
+
+1. How to bucket users into experiment's variation(s)
+2. How to track conversion events
+3. How to view the experiment’s results
+
+## Optimizely Full Stack Overview
+
+Optimizely Full Stack allows developers to run experiments anywhere in their code! The C# SDK provides the core components to run a full stack experiment with Optimizely. It handles aspects like bucketing, which is used to designate users to a specific experiment's variation, conversion tracking, and reporting via Optimizely’s [Stats Engine](https://www.optimizely.com/statistics/).  
+
+* View the [C# Getting Started Guide](https://developers.optimizely.com/x/solutions/sdks/getting-started/index.html?language=csharp)
+
+* View the reference [documentation](https://developers.optimizely.com/x/solutions/sdks/reference/index.html?language=csharp).
+
+* Latest [C# SDK](https://github.com/optimizely/csharp-sdk)
+
+## Demo App
+
+This example App simulates an online retailer testing the effects of sorting products by price vs name.
+
+Using the instructions below, you can run the app locally and mimic bucketing website visitors by selecting a users from "Select Visitor" page. For example, the user ID “Mike” would simulate a unique visitor and a specific variation would be selected for that unique visitor. The variation that is given to a specific unique visitor, such as "Mike", will be deterministic. This means as long as the experiment conditions remain the same, Mike will always get the same variation.
+ 
+<img src="https://github.com/optimizely/csharp-sdk/blob/master/OptimizelySDK.DemoApp/images/screenshot.png">
+
+### Deploying and Running the App
+
+1. Login or create an [Optimizely Account](https://app.optimizely.com/signin).
+2. Setup a C# project via the Optimizely dashboard. [Instructions](https://developers.optimizely.com/x/solutions/sdks/getting-started/?language=csharp)
+3. For the application to bucket users properly, the following values need to be defined in the experiment.
+	* Variation Keys: The experiment must contain at least two variations with keys "sort_by_price" and "sort_by_name".
+	* Event Key: The tracked event with key "AddToCart".
+4. Run the application OptimizelySDK.DemoApp.
+5. Click the "Demo App" link to go to the "Configuration" page.
+6. Provide ProjectId, ExperimentKey and EventKey values and hit Save to load the datafile of your project in ProjectConfigJson textbox. The Optimizely client is now initialized with your project configuration and is ready to serve the requests.
```

Comment:
You can remove 'EventKey' as this will misguide the user. 
Also provide a screenshot of this page

Reply:
I don't think that screenshots of our Demo App are required here. As the pages are pretty simple and the text above is good enough to guide the user about App usage.